testutil/e2e/e2e.go

```diff
@@ -770,7 +770,17 @@ func runE2E() {
 	lt.startJSONRPCProxy(jsonCTX)
 	lt.startJSONRPCProvider("http://127.0.0.1:1111", jsonCTX)
 	lt.startJSONRPCGateway(jsonCTX)
-	lt.checkJSONRPCGateway("http://127.0.0.1:3333/1", time.Second*30)
+	lt.checkJSONRPCGateway("http://127.0.0.1:3333/1", time.Minute*2)
```

Comment:
i see for other gateways we wait 30sec, why 2 minutes here?

Reply:
jsonrpc which uses ETH takes 64 blocks to boot which takes a long time.